src/main/java/com/nha/abdm/wrapper/common/RequestManager.java

```diff
@@ -45,4 +45,16 @@ public <T> ResponseEntity<GenericResponse> fetchResponseFromGateway(String uri,
         .toEntity(GenericResponse.class)
         .block();
   }
+
+  public ResponseEntity<Object> fetchResponseForPHR(
```

Comment:
does it mean that we are sending requests to PHR? if not, then lets rename this accordingly